core/config/docs/chains-evm.toml

```
@@ -263,6 +263,14 @@ TipCapDefault = '1 wei' # Default
 # (Only applies to EIP-1559 transactions)
 TipCapMin = '1 wei' # Default
 
+[EVM.GasEstimator.DAOracle]
+# OracleType refers to the oracle family this config belongs to (OP, Arbitrum, etc).
+OracleType = 'OP' # Example
```

Based on the toml OracleTypes, shouldn't the example be something like
```suggestion
OracleType = 'OPOracle' # Example
```

---

good catch thanks